<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiUpdate</class>
 <widget class="QDialog" name="uiUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../scc.qrc">
    <normaloff>:/images/logo16x16.png</normaloff>:/images/logo16x16.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_msg">
      <property name="text">
       <string>Message</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_download">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../scc.qrc"/>
 </resources>
 <connections/>
</ui>
